### DISCUSSION TITLE
How to use BRouter website

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#272] [Show and tell]
- Open the BRouter website:
  - https://brouter.de/brouter-test/
- Copy the content of a profile in the "Profile" tab and press "Apply"
- Use the tools on the right to analyze the route

<table>
<tr>
<td><img src="https://github.com/devemux86/cruiser/assets/3484020/feb13d03-1c51-4567-ba13-9c04788c9a5e"/></td>
<td><img src="https://github.com/devemux86/cruiser/assets/3484020/5785238d-0e9a-462e-8702-ecf676ad0e88"/></td>
<td><img src="https://github.com/devemux86/cruiser/assets/3484020/b9f23e5b-0910-435a-b5e1-41614e4beb99"/></td>
</tr>
</table>

---

BRouter website can export speed limits in GeoJSON format:
- Enable **turn instructions** in profile options

- Export in GeoJSON format (check **Waypoints**)

- Import the GeoJSON in Cruiser (**without calculation**)

![BRouter](https://github.com/devemux86/cruiser/assets/3484020/ec4b3906-3b8a-4b9d-aeba-0f72d494a408)

---

BRouter website can also export in GPX navigation format:
- Select **locus-style** in **turnInstructionMode**

- Export in GPX format (press **cloud** button)

-  Import the GPX in Cruiser (**without calculation**)

![BRouter web client](https://github.com/devemux86/cruiser/assets/3484020/f5ea6c19-aae7-4990-b0f4-a4b84b6753da)

**Comment by DolomitesTom** [comment]
How do I easily extract or export the current settings from the internal profile in the app in order to use it in the brouter website?

I don't remember in which file and path are the internal profile settings stored. When you don't use it for a long time, a function in the app would be good to to export the settings into a file.

**Reply by devemux86** [reply]
If you mean to export the BRouter profile from Cruiser **with** your changes and load it on BRouter website,
this could probably be implemented, I will need to check the process.

**Reply by DolomitesTom** [reply]
Yes that's what I've meant
